calico_containers/calicoctl.py

```diff
@@ -1648,275 +84,152 @@ def permission_denied_error(conn_error):
     return True
 
 
-def print_container_not_in_calico_msg(container_name):
-    """
-    Display message indicating that the supplied container is not known to
-    Calico.
-    :param container_name: The container name.
-    :return: None.
-    """
-    print_paragraph("Container %s is unknown to Calico." % container_name)
-    print_paragraph("Use `calicoctl container add` to add the container "
-                    "to the Calico network.")
-
-
-def print_paragraph(msg):
-    """
-    Print a fixed width (80 chars) paragraph of text.
-    :param msg: The msg to print.
-    :return: None.
-    """
-    print "\n".join(textwrap.wrap(msg, width=80))
-    print
-
-
-def parse_ports(ports_str):
-    """
-    Parse a string representing a port list into a list of ports and
-    port ranges.
-
-    Returns None if the input is None.
+def validate_arguments(arguments):
```

Comment:
I would much prefer that validate_argument was split between the sub command modules.